src/styles/main.scss

```diff
@@ -235,6 +235,28 @@ button {
   display: none;
 }
 
+table {
+  background: #fff;
+  border: solid 1px #ddd;
+  margin-bottom: 1.25rem;
+  width: 100%;
+
+  thead,
+  tfoot {
+    background: #f5f5f5;
```

Comment:
bold would be super nice here, I think! what's your opinion?

Reply:
in most browsers `th` has `font-weight: bold;`